website/docs/upgrades/1.0_to_1.1/automatic_schema_matching.md

```diff
@@ -92,3 +98,78 @@ cs.store(group="db",
 </div>
 </div>
 
+</details>
+
+### Rename the config file
+Steps:
+1. Rename the config file. Common choices are `custom_` or `my_` prefix, e.g. `custom_mysql.yaml`.
+2. Add the schema to the Defaults List of the extending config file.
+
+
+<details><summary>Click to expand</summary>
+
+#### Hydra 1.0
+<div className="row">
+<div className="col col--6">
+
+```yaml title="db/mysql.yaml"
+# @package _group_
+host: localhost
+port: 3306
+
+
+
+
+
+
+```
+</div>
+<div className="col col--6">
+
+```python title="Schema for db/mysql.yaml"
+@dataclass
+class MySQLConfig:
+    host: str
+    port: int
+
+cs = ConfigStore.instance()
+cs.store(group="db",
+         name="mysql", 
+         node=MySQLConfig)
+```
+</div>
+</div>
+
+#### Hydra 1.1
+<div className="row">
+<div className="col col--6">
+
+```yaml title="db/custom_mysql.yaml" {1,2}
```

Comment:
It would be good to highlight the new name. I'm not sure it can be done through Docusaurus, but something like this could work:
```suggestion
```yaml title="db/custom_mysql.yaml <= NEW NAME!" {1,2}
```